Data_Structures_Questions.md

```diff
@@ -31,23 +31,25 @@ Answer the following questions for each of the data structures you implemented a
 ## Doubly Linked List
 
 1. What is the runtime complexity of `ListNode.insert_after`?
-
+O(1)
 2. What is the runtime complexity of `ListNode.insert_before`?
-
+O(1)
 3. What is the runtime complexity of `ListNode.delete`?
-
+O(1)
 4. What is the runtime complexity of `DoublyLinkedList.add_to_head`?
-
+O(1)
 5. What is the runtime complexity of `DoublyLinkedList.remove_from_head`?
-
+O(1)
 6. What is the runtime complexity of `DoublyLinkedList.add_to_tail`?
-
+O(1)
 7. What is the runtime complexity of `DoublyLinkedList.remove_from_tail`?
-
+O(1)
 8. What is the runtime complexity of `DoublyLinkedList.move_to_front`?
-
+O(n)
```

Comment:
Why? this method receives the node we wish to move. We don't need to search for it.